tsl/test/expected/compression.out

```diff
@@ -80,10 +80,9 @@ select compress_chunk( '_timescaledb_internal._hyper_1_2_chunk');
 select tgname , tgtype, tgenabled , relname
 from pg_trigger t, pg_class rel
 where t.tgrelid = rel.oid and rel.relname like '_hyper_1_2_chunk' order by tgname;
-             tgname              | tgtype | tgenabled |     relname      
----------------------------------+--------+-----------+------------------
- compressed_chunk_insert_blocker |      7 | O         | _hyper_1_2_chunk
-(1 row)
+ tgname | tgtype | tgenabled | relname 
```

Comment:
I guess we should now remove this whole check if the insert blocker is no longer needed?